app/models/topic.rb

```diff
@@ -22,14 +24,9 @@ def slug
   end
 
   def guide_content_ids
-    ids = tree.map do |grouping|
-      grouping['guides'].map do |guide_id|
-        Integer(guide_id)
-      end
-    end.flatten.uniq
-    Guide
-      .where(id: ids)
-      .pluck(:content_id)
+    topic_sections.map do |topic_section|
+      topic_section.guides.map(&:content_id)
+    end.flatten
```

Comment:
If we defined a `has_many :guides, through: :topic_sections` then this method could become `guides.map(&:content_id)` which would be nice.

It looks like the `uniq` is gone - do we no longer need it here?